packages/components/src/internal/components/forms/input/TextInput.tsx

```diff
@@ -90,6 +90,9 @@ export class TextInput extends DisableableInput<TextInputProps, TextInputState>
         const { label, queryColumn, showLabel, allowDisable, addLabelAsterisk, renderFieldLabel } = this.props;
         const { isDisabled } = this.state;
 
+        if (!showLabel)
+            return null;
```

Comment:
@labkey-susanh sorry for the late update on this, but I just noticed that these two lines of code are causing display issues with the details panel edit mode (see here from LKB Registry molecules details panel edit, but also seen with samples detail edit ). When I comment out these two lines, the form inputs placement returns to the expected place.
<img width="917" alt="Screen Shot 2022-12-15 at 12 55 21 PM" src="https://user-images.githubusercontent.com/6411206/207944346-f5383eed-3043-408c-be9a-d1a3bb6f6037.png">

Reply:
Thanks for the heads up. Fixed.